composer.json

```diff
@@ -14,13 +14,13 @@
   },
   "require": {
     "php": ">=7.2",
-    "doctrine/lexer": "^1.2",
+    "doctrine/lexer": "^1.2|^2",
```

Comment:
Is there a reason why not to require v2 only?

Reply:
Lexer is a transitive dependency in many projects, shared by more than one library. Dropping v1 right away might cause conflicts downstream at the moment.